Describe the Form - Overview demo (#658)
DevExpress/devextreme-demos: JSDemos/Demos/Form/Overview/description.md

The Form component represents fields of a data object as a collection of label-editor pairs. To create a data entry UI, assign an object to the [formData](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#formData) property. The Form creates a simple item (a label-editor pair) for each field in the **formData** object.

You can use the editors on the right to modify the following properties:     
- [labelLocation](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#labelLocation)

> Specifies whether to display item labels on top or on the left

Reviewer: It's unclear on top of what the label is displayed.

```suggestion
Specifies whether to display item labels above the editor or to the left of it.
```

- [colCount](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#colCount)
Specifies the number of columns. Assign a number to this property to make the amount of columns fixed. Assign *"auto"* to make it adapt to the screen width.
- [minColWidth](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#minColWidth)
Specifies the minimum column width. Use this property when the **colCount** property's value is *"auto"*.
- [width](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#width)
Specifies the Form component's width.
- [readOnly](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#readOnly)
Makes the Form editors read-only.
- [showColonAfterLabel](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#showColonAfterLabel)

> Specifies whether or not the Form displays a colon after a label. 

Reviewer: Property descriptions are inline with property names:
![изображение](https://user-images.githubusercontent.com/18551316/111273096-72fccb80-8644-11eb-962a-70beb5913666.png)

It'll also be great if we add an empty line between each list item to give them more space.

> The Form component builds data entry UI for an object assigned to the [formData](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#formData) property. The component displays and aligns label-editor pairs for each field in the bound object.

Reviewer 2: ```suggestion
The Form component builds a data entry UI for an object assigned to the [formData](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#formData) property. The component displays and aligns label-editor pairs for each field in the bound object.
```

> The Form component represents fields of a data object as a collection of label-editor pairs. To create a data entry UI, assign an object to the [formData](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#formData) property. The Form creates a simple item (a label-editor pair) for each field in the **formData** object.

Reviewer 3: `Represents` is a commonly misused word, it's best to avoid it. 

There are three sentences that somewhat repeat information (for example, you mention "label-editor pairs" twice). I changed that to two. 

```suggestion
The Form component builds data entry UI for an object assigned to the [formData](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#formData) property. The component displays and aligns label-editor pairs for each field in the bound object.
```

You can use the editors on the right to modify the following properties:     
- [labelLocation](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#labelLocation)    

> Specifies whether to display item labels above the editor or to the left of it.

Reviewer 3: Labels - plural, editor - singular. Reads weird to me. 

```suggestion
Specifies whether to display labels above or to the left of corresponding editors.
```

- [colCount](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#colCount)  

> Specifies the number of columns in the layout. To build an adaptive layout where column count depends on container width, set this property's value to *"auto"*.

Reviewer 2: ```suggestion
Specifies the number of columns in the layout. To build an adaptive layout where the column count depends on the container width, set this property's value to *"auto"*.
```

> Specifies the number of columns. Assign a number to this property to make the amount of columns fixed. Assign *"auto"* to make it adapt to the screen width.

Reviewer 3: 'amount' is used with uncountable nouns. 
First and second sentences are really similar - I think one is enough. 

```suggestion
Specifies the number of columns in the layout. To build an adaptive layout where column count depends on container width, set this property's value to *"auto"*.
```

- [minColWidth](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#minColWidth)    
Specifies the minimum column width. Use this property when the **colCount** property's value is *"auto"*.

- [width](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#width)    
Specifies the Form component's width.

- [readOnly](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#readOnly)  
Makes the Form editors read-only.

- [showColonAfterLabel](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#showColonAfterLabel)    

> Specifies whether or not the Form displays a colon after a label. 

Reviewer 2: ```suggestion
Specifies whether the Form displays a colon after a label. 
```

Reviewer 2: There is no need to add 'or not'
